document-store/src/main/java/org/hypertrace/core/documentstore/expression/BinaryOperator.java

```diff
@@ -0,0 +1,11 @@
+package org.hypertrace.core.documentstore.expression;
+
+public enum BinaryOperator {
+  ADD,
+  SUBTRACT,
+  MULTIPLY,
+  DIVIDE,
+  REMINDER,
```

Comment:
Guessing this is supposed to be remainder. It's easier to add things than remove them, so I would start with the bare minimum here. Maybe just the first four?

Reply:
Nice catch. Sure.